projection/bridge_activity/bridge_pending_activity/bridge_pending_activity.go

```diff
@@ -345,9 +361,11 @@ func (projection *BridgePendingActivity) HandleEvents(height int64, events []eve
 			}
 
 		} else if cronosSendToIBCCreatedEvent, ok := event.(*event_usecase.CronosSendToIBCCreated); ok {
```

Comment:
Actaully are we able to remove this Cronos specific event because after this PR we will support multiple custom chain.

Reply:
Not exactly because this is a special type of bridge transaction on Cronos chain.

Some background story is that on Cronos you can think of it has the EVM layer and Cosmos layer.

Since most users are interacting with the EVM layer, so IBC transfers, which is Cosmos layer, is triggered by sending a ETH Tx. That's why we need this special event for detecting this kind of IBC transfers.

Reply:
If we change to `ethereumSendToIBCCreatedEvent` or `evmSendToIBCCreatedEvent`, do you think it is a proper but generic naming?